neo/io/nwbio.py

```diff
@@ -441,8 +453,14 @@ def write_all_blocks(self, blocks, **kwargs):
                 raise Exception("Writing to NWB requires an annotation 'session_start_time'")
         self.annotations = {"rec_datetime": "rec_datetime"}
         self.annotations["rec_datetime"] = blocks[0].rec_datetime
-        # todo: handle subject
+        self.annotations = {"subject": "subject"}
         nwbfile = NWBFile(**annotations)
+        if "subject" not in annotations:
+            nwbfile.subject = Subject(subject_id="subject_id",
+                                      age="P0D",  # Period x days old
+                                      description="no description",
+                                      species="Mus musculus",  # by default
+                                      sex="U")
```

Comment:
These are default placeholders, right? Maybe it would be good to explicitely use dummy labels here, e.g. `empty_neo_subject_id` or `neo_dummy_subject_id`?